frontend/src/views/web/challenge/update-submission-metadata.html

```diff
@@ -1,35 +1,189 @@
+    <div class="col s12 m12">
+      <div class="ev-md-container ev-card-body update-profile-card">
+        <form
+          name="updateSubmissionMetaDataForm"
+          ng-submit="challenge.updateSubmissionMetaData(updateSubmissionMetaDataForm.$valid)"
+        >
+          <div class="pass-title">Update Submission Details</div>
+          <div class="input-field align-left">
+            <input
+              id="methodName"
+              name="methodName"
+              type="text"
+              class="text-dark-black dark-autofill w-400"
+              ng-model="challenge.method_name"
+              value="{{challenge.method_name}}"
+              focus-if="challenge.method_name"
+            />
+            <span class="form-icon form-icon-dark">
+              <i class="fa fa-pencil"></i>
+            </span>
+            <label class="active" for="methodName">Method Name</label>
+          </div>
+          <div class="input-field align-left">
+            <textarea
+              id="methodDesc"
+              name="methodDesc"
+              type="text"
+              class="materialize-textarea"
+              ng-model="challenge.method_description"
+              value="{{challenge.method_description}}"
+              focus-if="challenge.method_description"
+            ></textarea>
+            <span class="form-icon form-icon-dark">
+              <i class="fa fa-pencil"></i>
+            </span>
+            <label for="methodDesc">Method Description</label>
+          </div>
+          <div class="input-field align-left">
+            <input
+              id="projectUrl"
+              name="projectUrl"
+              type="text"
+              class="text-dark-black dark-autofill w-400"
+              ng-model="challenge.project_url"
+              value="{{challenge.project_url}}"
+              focus-if="challenge.project_url"
+            />
+            <span class="form-icon form-icon-dark">
+              <i class="fa fa-pencil"></i>
+            </span>
+            <label for="projectUrl">Project Url</label>
+          </div>
+          <div class="input-field align-left">
+            <input
+              id="publicationUrl"
+              name="publicationUrl"
+              type="text"
+              class="text-dark-black dark-autofill w-400"
+              ng-model="challenge.publication_url"
+              value="{{challenge.publication_url}}"
+              focus-if="challenge.publication_url"
+            />
+            <span class="form-icon form-icon-dark">
+              <i class="fa fa-pencil"></i>
+            </span>
+            <label for="publicationUrl">Publication Url</label>
+          </div>
+          <div
+            ng-if="challenge.submissionMetaDataTemp != null"
+            ng-repeat="attribute in challenge.submissionMetaDataTemp"
+          >
+            <div
+              ng-if="attribute.type == 'text'"
+              class="input-field align-left"
+            >
```

Comment:
don't put `>` on a newline. If the whole `<div ng-if="attribute.type == 'text'" class="input-field align-left">` can fit in one line please do that